internal/db/timestamp/timestamp.go

```diff
@@ -22,5 +22,5 @@ func Now() *Timestamp {
 
 // AsTime converts x to a time.Time.
 func (x *Timestamp) AsTime() time.Time {
-	return x.AsTime()
+	return x.Timestamp.AsTime()
```

Comment:
Why the change?

Reply:
`x.AsTime()` keeps calling this same function iteratively leading to stack overflow